queries.sql

```diff
@@ -8,3 +8,42 @@ SELECT (name, escapes_attempts) FROM animals WHERE weight_kg >10.5;
 SELECT * FROM animals WHERE neutered IS true;
 SELECT * FROM animals WHERE name IN ('Agumon');
 SELECT * FROM animals WHERE weight_kg >=10.4 AND weight_kg <=17.3;
+
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species='pokemon' WHERE species='unspecified';
```

Comment:
The default value for species if not specified is NULL in the database. Therefore, your query `WHERE species='unspecified'` is invalid.

- You can optimize your query by using `WHERE species IS NULL`.